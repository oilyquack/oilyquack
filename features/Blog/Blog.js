import React from "react";

const Blog = () => (
  <>
    <h1>Coming soon</h1>
  </>
);

export default Blog;
