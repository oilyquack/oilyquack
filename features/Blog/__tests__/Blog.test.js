import { render } from "@testing-library/react";
import React from "react";

import Blog from "../Blog";

describe("features : Blog", () => {
  it("renders correctly", () => {
    const { container } = render(<Blog />);
    expect(container).toMatchSnapshot();
  });
});
