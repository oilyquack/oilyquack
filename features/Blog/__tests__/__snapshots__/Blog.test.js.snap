// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`features : Blog renders correctly 1`] = `
<div>
  <h1>
    Coming soon
  </h1>
</div>
`;
